bot/tests/test_backend.py

```diff
@@ -204,3 +204,34 @@ def test_publication_failures(
             "fix": None,
         }
     ]
+
+
+def test_publication_failures_total(
```

Comment:
We should also add a test which runs the `run` function of the workflow and, even if the backend fails, it publishes some issues